### DISCUSSION TITLE
lots of questions

### DISCUSSION THREAD

**Discussion by Louistam888** [vikejs/vike#1723] [Discussion]
Hi there, I’m very new to this so I appreciate your patience.

I’m looking to render my existing vite react project as an SSG with vike.

A few questions I have:
1.) Will existing CSS libraries I have continue to work fine?
2.) Am I doing away with my router structure? The docs simultaneously say to use vike’s router and that react’s router works fine
3.) I am utterly lost with the documentation. The section on adding vike to an existing project simply says to install, add some boilerplate, then create a +page file. I’m sure there’s a lot more involved in integration than just that

**Comment by brillout** [comment]
1) Sure, why wouldn't they?
2) What isn't clear about https://vike.dev/react-router?
3) What isn't clear about https://vike.dev/add?
